lib/addons/vpc-cni/index.ts

```diff
@@ -450,7 +481,12 @@ function populateVpcCniConfigurationValues(props?: VpcCniAddOnProps): Values {
       WARM_PREFIX_TARGET: props?.warmPrefixTarget,
     },
     enableNetworkPolicy: JSON.stringify(props?.enableNetworkPolicy),
-    enableWindowsIpam: JSON.stringify(props?.enableWindowsIpam)
+    enableWindowsIpam: JSON.stringify(props?.enableWindowsIpam),
```

Comment:
minor: is there a reason to stringify this here? the code on line 495 should take care of that, if it does not, please let me know. Ideally, I want to have a consistent approach to extend this structure.

Reply:
The code in line 459 in deed does NOT take care for properties on the toplevel config object.

To avoid mixing up different things in one PR I created #1048 and added a more detailed description there.